### ISSUE TITLE
UDF with a variable number of arguments

### ISSUE THREAD

**Issue by JohnnySG** [cunnane/xloil#19]
Hi Steven,

Thank you for your great work!
I want to create a UDF with a variable number of arguments:

```
@xloil.func
def add(*numbers):
    total = 0
    for num in numbers:
        total += num
    return total
```

However, I got the error below:
> [18:10:04] [error] Failed determing spec for 'add': Traceback (most recent call last):
>   File "C:\Users\JohnnyS\miniconda3\lib\site-packages\xloil\register.py", line 356, in decorate
>     func_args, return_type = Arg.full_argspec(fn)
>   File "C:\Users\JohnnyS\miniconda3\lib\site-packages\xloil\func_inspect.py", line 88, in full_argspec
>     args = [cls.from_signature(name, param) for name, param in params.items()]
>   File "C:\Users\JohnnyS\miniconda3\lib\site-packages\xloil\func_inspect.py", line 88, in <listcomp>
>     args = [cls.from_signature(name, param) for name, param in params.items()]
>   File "C:\Users\JohnnyS\miniconda3\lib\site-packages\xloil\func_inspect.py", line 78, in from_signature
>     raise Exception(f"Unhandled argument '{name}' with type '{kind}'")
> Exception: Unhandled argument 'numbers' with type 'VAR_POSITIONAL'

Could you please give me a example!

Thanks and Regards,
Johnny

**Comment by cunnane** [comment]
This feature (i.e. using `*args`) isn't currently supported, though I could add it.  It would look mildly ugly in the function wizard as it would be registered as a function with 200 args, rather than the way the built-in functions expand the function wizard as more arguments are added.

**Comment by cunnane** [comment]
This is now available in xlOil v0.16
